django/db/models/query.py

```diff
@@ -319,15 +319,13 @@ def aggregate(self, *args, **kwargs):
         if self.query.distinct_fields:
             raise NotImplementedError("aggregate() + distinct(fields) not implemented.")
         for arg in args:
-            # The default_alias property may raise a TypeError, so we use
-            # a try/except construct rather than hasattr in order to remain
-            # consistent between PY2 and PY3 (hasattr would swallow
-            # the TypeError on PY2).
+            # The default_alias property may raise a TypeError. Use try/except
```

Comment:
I'd say something like:
```
# The default_alias property raises TypeError if .... or AttributeError if ...
```
I'm not sure what the second sentence means. It seems like you're saying `hasattr(arg, 'default_alias')` raises an exception if `'default_alias'` isn't a string.